src/tests/obj_ctl.c

```diff
@@ -487,7 +372,7 @@ main(int argc, char *argv[])
 {
 	int	rc;
 
-	if (argc < 2)
+	if (argc ==2 )
```

Comment:
(style)  spaces required around that '==' (ctx:WxV)
(style)  space prohibited before that close parenthesis ')'